### ISSUE TITLE
Redux v4 compatibility

### ISSUE THREAD

**Issue by mpeyper** [ioof-holdings/redux-subspace#87]
[Redux v4 has been officially released now](https://github.com/reactjs/redux/releases/tag/v4.0.0) so we need to ensure that all of our packages are still compatible.

From my understanding, there aren't many breaking changes so it should hopefully just be a case of bumping the version and running the tests to ensure it all still works.

The `redux` peer dependency should continue to support `^3.0.0` so it will need changing to `^3.0.0 || ^4.0.0` (or however else it's possible to define it that allows consumers to user v3 or v4 without getting a warning).

**Comment by majo44** [comment]
Hi @mpeyper can we make dev/alfa/beta release from master. We are using observables and we want to migrate to Rv4 and Ov1

**Comment by mpeyper** [comment]
Hi @majo44, yes we should do this.  I will look into it tomorrow (it's 10pm my time).

**Comment by majo44** [comment]
Any update ? :)

**Comment by mpeyper** [comment]
Kind of. I've got some excuses why it's not out yet... I went to do it, bit thought I'd take a "quick" look at what was left for redux v4 compatibility in the other packages so I could do a prerelease for all of them.  Long story short, I started pulling a thread and now I'm a [few commits deep](https://github.com/mpeyper/redux-subspace/commits/master) into a redux-loop upgrade. I hope it doesn't take too much longer.

Sorry for the delay.
